packages/block-library/src/cover/edit/poster-image.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * WordPress dependencies
+ */
+import { MediaUpload, MediaUploadCheck } from '@wordpress/block-editor';
+import {
+	Button,
+	BaseControl,
+	__experimentalHStack as HStack,
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { useRef } from '@wordpress/element';
+import { useInstanceId } from '@wordpress/compose';
+
+const COVER_POSTER_ALLOWED_MEDIA_TYPES = [ 'image' ];
+
+function PosterImage( { poster, setAttributes } ) {
+	const posterImageButton = useRef();
+	const instanceId = useInstanceId( PosterImage );
+	const videoPosterDescription = `cover-block__poster-image-description-${ instanceId }`;
```

Comment:
```suggestion
	const descriptionId = useInstanceId( PosterImage, 'cover-block__poster-image-description' );
```

Suggestion: The `useInstanceId` hook accepts a second `prefix` parameter. Removes the need for manual string contact.

Nothing wrong with current versions, that's why we don't refactor similar instances. Just something to keep in mind for the future.

Reply:
Thanks for pointing this out. Looks like I missed this 🤦‍♂️

I'd still like to refactor as it looks much cleaner.